lib/client.js

```diff
@@ -189,6 +189,60 @@ class GoogleOauth2Client {
     return result;
   }
 
+  async writeColumnToSpreadsheet(opts) {
+    this.logger.debug('Trying to write a column to spreadsheet...');
+    const { spreadsheetId } = this.configuration;
+    const { range, values } = opts;
+    const sheets = google.sheets({ version: sheetsVersion, auth: this.client });
+    const result = await sheets.spreadsheets.values.append({
+      spreadsheetId,
+      range,
+      valueInputOption: 'RAW',
+      resource: {
+        majorDimension: 'COLUMNS',
+        values: [values],
+      },
+    });
+    this.logger.debug('Spreadsheet column written successfully');
+    return result;
+  }
+
+  async updateSpreadsheetRow(opts) {
+    this.logger.debug('Trying to update a row in spreadsheet...');
+    const { spreadsheetId } = this.configuration;
+    const { range, values } = opts;
+    const sheets = google.sheets({ version: sheetsVersion, auth: this.client });
+    const result = await sheets.spreadsheets.values.update({
+      spreadsheetId,
+      range,
+      valueInputOption: 'RAW',
+      resource: {
+        majorDimension: 'ROWS',
+        values: [values],
+      },
+    });
+    this.logger.debug('Spreadsheet row updated successfully');
+    return result;
+  }
+
+  async updateSpreadsheetColumn(opts) {
+    this.logger.debug('Trying to update a column to spreadsheet...');
+    const { spreadsheetId } = this.configuration;
+    const { range, values } = opts;
+    const sheets = google.sheets({ version: sheetsVersion, auth: this.client });
+    const result = await sheets.spreadsheets.values.update({
+      spreadsheetId,
+      range,
+      valueInputOption: 'RAW',
+      resource: {
+        majorDimension: 'COLUMNS',
+        values: [values],
+      },
+    });
+    this.logger.debug('Spreadsheet column updated successfully');
+    return result;
+  }
```

Comment:
Same here (looks familiar to `writeColumnToSpreadsheet` and `updateSpreadsheetColumn`)